roles/api-environment/defaults/main.yml

```diff
@@ -15,7 +15,9 @@ disable_requiretty_for_api_user: "Defaults:{{ api_user }} !requiretty"
 jenkins_restart_script: "jenkins-restart.sh"
 restart_container_script: "restart-container.sh"
 restart_container_path: "{{ api_home }}/{{ restart_container_script }}"
-container_health_interval_seconds: "60"
+container_health_interval_seconds: 60
+container_health_timeout_seconds: 30
```

Comment:
This health timeout interval is Docker's default, so nothing should be being changed here.